decidim-core/app/presenters/decidim/admin_log/static_page_presenter.rb

```diff
@@ -30,6 +30,19 @@ def diff_fields_mapping
       def resource_presenter
         @resource_presenter ||= Decidim::AdminLog::StaticPageResourcePresenter.new(action_log.resource, h, action_log.extra["resource"])
       end
+
+      def i18n_labels_scope
+        "activemodel.attributes.static_page"
+      end
```

Comment:
Is this expected to be here?

Reply:
Yes, it is. It's overwriting a method in the `BasePresenter`, I forgot it in the previous PR and didn't want to create a new one for this :(